packages/api/test/e2e/api/promise-queries.spec.ts

```diff
@@ -234,29 +144,135 @@ describeE2E()('Promise e2e queries', (wsUrl): void => {
       expect(size.toNumber()).toBeGreaterThanOrEqual(0);
     });
   });
+});
+
+// The following tests require a development node with the default funded testing accounts
+describeE2E({
```

Comment:
I would probably suggest just having one `describeE2E` block per file. That way we categorize things slightly better?

Reply:
+1

Reply:
Yes, true, but naming the files could get quite hard then, but I'll try to think of something.

Reply:
I split this file into three separate ones.